components/mpas-ocean/src/shared/mpas_ocn_vmix_cvmix.F

```diff
@@ -169,6 +172,11 @@ subroutine ocn_vmix_coefs_cvmix_build(meshPool, statePool, forcingPool, err, tim
 
       real (kind=RKIND) :: langmuirEnhancementFactor, langmuirNumber
 
+      real (kind=RKIND) :: latCellDeg, latCellDegAbs
+      real (kind=RKIND),allocatable,save :: bdlatfn(:)
+      logical :: debug_bdlatfn = .false.
```

Comment:
@apcraig: do you feel like we still need to keep the debugging option?

Reply:
maybe it would be good to have it, for when we test this PR.

Reply:
It doesn't hurt to leave it there, but have it off by default.  We're already talking about making changes to the function, so I think it's useful to leave it in.  But if others want it out, I can take it out.  It seems to be the only way to validate the implementation.